*: Standardize install invocation as per #276

tox-wtf opened the pull request:
> [!WARNING]
> This is not tested, and is thus marked as a draft until we confirm no regressions are introduced by these changes.

This commit standardizes all invocations of `install`[^1]. In so doing, I noticed a few places where instructions could be condensed, so other commands are also modified.

[^1]: As found with `find . -name '*.xml' | xargs grep -Hn 'install -'`

This commit does not consider the `install-*` XML entities, and does not address the changes that need to be made to `/CONTRIBUTING.md`. Nor does it consider whitespace formatting in commands. These issues will be addressed in future commits.

tox-wtf committed:
*: Standardize install invocation as per #276

> [!WARNING]
> This is not tested, and is thus marked as a draft until we confirm no
> regressions are introduced by these changes.

This commit standardizes all invocations of `install`[^1]. In so doing, I
noticed a few places where instructions could be condensed, so other
commands are also modified.

[^1]: As found with `find . -name '*.xml' | xargs grep -Hn 'install -'`

This commit does not consider the `install-*` XML entities, and does not
address the changes that need to be made to `/CONTRIBUTING.md`. Nor does
it consider whitespace formatting in commands. These issues will be
addressed in future commits.

tox-wtf committed:
*: Fixups for install standardization

This commit addresses whitespace formatting, makes a few corrections,
and simplifies a few more commands.

tox-wtf committed:
*: Replace tabs with spaces

tox-wtf committed:
CONTRIBUTING: Add style guide

tox-wtf commented:
I imagine it would be simpler and more future-proof to use a style guide for `install` (and possibly other commands) rather than an XML entity. I've written a stub one and linked it in `/CONTRIBUTING.md`. This also has the advantage of moving all styling-related contributing guidelines to one place.

I'd appreciate some feedback on what's currently included there.

tox-wtf committed:
STYLE-GUIDE: Add section for source code

tox-wtf committed:
CONTRIBUTING: Rework

Zeckmathederg commented:
I don't yet have the time to look at this. Don't know when I will. But it should be soon.

Zeckmathederg commented:
Took a look at this and it looks good so far.

tox-wtf committed:
Install icons to the correct location

Install application icon files to the correct location[^1]. This only
tackles alacritty and htop, though other packages may also install icons
to the pixmaps directory.

[^1]: As best I can gauge, `/usr/share/icons` should be preferred to
`/usr/share/pixmaps`:
https://askubuntu.com/questions/477445/confused-about-app-icons-folders,
https://github.com/spyder-ide/spyder/issues/6188

tox-wtf committed:
libime: Simplify submodule unpacking

Zeckmathederg commented:
Thanks for removing that line in https://github.com/glfs-book/slfs/pull/452/commits/1105424c7baef57bd0ecb952606794206fef5530. I didn't want to touch it just to be safe since I was in a rush when altering the commands.

tox-wtf committed:
rpmextract: Fix typo

tox-wtf committed:
GNAT: Add missing role="root"

tox-wtf commented:
Package testing:
- [ ] FreeBee
- [ ] idle
- [ ] p20-emu
- [ ] s4-3b1-pc7300
- [ ] XHomer
- [x] Fuse2
- [ ] libime
- [ ] FluidSynth
- [x] htop
- [x] jstest-sdl
- [x] rpmextract
- [ ] GNAT
- [ ] Unlambda
- [ ] tealdeer
- [ ] Mesa-Demos
- [ ] Fcitx5
- [ ] Alacritty
- [ ] Motif
- [ ] s-nail
- [ ] star from schilytools
- [ ] CDE

tox-wtf committed:
Alacritty: Simplify building man pages

tox-wtf committed:
Alacritty: Fix double slash in path

zeckma-tech commented on graph/fcitx/fcitx5.xml:
```diff
@@ -123,9 +123,8 @@ cp -rv doc/html/* /usr/share/doc/fcitx5-&fcitx5-version;</userinput></screen>
-cp /usr/share/applications/org.fcitx.Fcitx5.desktop \
-         ~/.config/autostart</userinput></screen>
+<screen><userinput>install -vDm644 /usr/share/applications/org.fcitx.Fcitx5.desktop -t ~/.config/autostart/
+</userinput></screen>
```
Ensure `</userinput></screen>` appears at the same line as `-t ~/.config/autostart/`.

tox-wtf replied:
I'll fix other occurrences of this issue as well, as found with `rg '^</userinput></screen>'`.

tox-wtf committed:
Fcitx5: Formatting

tox-wtf committed:
*: Ensure </userinput></screen> doesn't exist on its own line

tox-wtf committed:
Merge branch 'trunk' into install
